Extensions/ArtifactEngine/package.json

```diff
@@ -18,7 +18,7 @@
   "dependencies": {
     "handlebars": "4.0.10",
     "minimatch": "3.0.2",
-    "vsts-task-lib": "^2.1.0",
+    "vsts-task-lib": "2.2.1",
```

Comment:
this require 2.2.1 version of task lib?

Reply:
Yes we require 2.2.1, this allows to set multiple resource file for the task.